contracts/sdk/bubblegum/src/convenience.ts

```diff
@@ -76,3 +77,40 @@ export async function getCreateTreeIxs(
     );
     return [allocAccountIx, initGummyrollIx];
 }
+
+export function computeMetadataArgsHash(mintIx: TransactionInstruction) {
+    const metadataArgsBuffer = mintIx.data.slice(8)
+    return keccak_256.digest(metadataArgsBuffer);
+}
+
+export function computeDataHash(sellerFeeBasisPoints: number, mintIx?: TransactionInstruction, metadataArgsHash?: number[]) {
```

Comment:
@ngundotra moved the `computeDataHash` and `computeCreatorHash` methods here. agree, way cleaner!